examples/medical-center/generated/persist_client.bal

```diff
@@ -50,62 +50,61 @@ public client class MedicalCenterClient {
         }
         self.dbClient = dbClient;
         self.persistClients = {
-            medicalitem: check new (self.dbClient, self.metadata.get(MEDICAL_ITEM)),
-            medicalneed: check new (self.dbClient, self.metadata.get(MEDICAL_NEED))
+            medicalitems: check new (self.dbClient, self.metadata.get(MEDICAL_ITEM)),
+            medicalneeds: check new (self.dbClient, self.metadata.get(MEDICAL_NEED))
```

Comment:
```suggestion
            [MEDICAL_ITEM]: check new (self.dbClient, self.metadata.get(MEDICAL_ITEM)),
            [MEDICAL_NEED]: check new (self.dbClient, self.metadata.get(MEDICAL_NEED))
```

Reply:
Fixed in [d0b65f0](https://github.com/ballerina-platform/module-ballerina-persist/pull/177/commits/d0b65f00336f1917043ae3b82e35b0c405656d9f)